src/app/api.py

```diff
@@ -6,6 +12,31 @@
 from src.app.monitor import api_router as monitor_router
 from src.jbi.router import api_router as jbi_router
 
+settings = get_settings()
+
+logging_config = {
+    "version": 1,
+    "formatters": {
+        "json": {
+            "()": "dockerflow.logging.JsonLogFormatter",
+            "logger_name": "jbi",
+        },
+    },
+    "handlers": {
+        "console": {
+            "level": settings.log_level.upper(),
+            "class": "logging.StreamHandler",
+            "formatter": "json",
+            "stream": sys.stdout,
+        }
+    },
+    "loggers": {
+        "request.summary": {"handlers": ["console"], "level": "INFO"},
+    },
+}
+
+logging.config.dictConfig(logging_config)
```

Comment:
```suggestion
@app.on_event("startup") 
def startup_event():
    logging_config = {
        "version": 1,
        "formatters": {
            "json": {
                "()": "dockerflow.logging.JsonLogFormatter",
                "logger_name": "jbi",
            },
        },
        "handlers": {
            "console": {
                "level": settings.log_level.upper(),
                "class": "logging.StreamHandler",
                "formatter": "json",
                "stream": sys.stdout,
            }
        },
        "loggers": {
            "request.summary": {"handlers": ["console"], "level": "INFO"},
        },
    }
    
    logging.config.dictConfig(logging_config)
```

Reply:
Something like this maybe? Then the internals could be extracted to `log.py` and then called on this startup method.

What are your thoughts on that?